pref/battery/battery_windows.go

```diff
@@ -0,0 +1,80 @@
+package battery
+
+import (
+	"golang.org/x/sys/windows"
+	"unsafe"
+)
+
+var (
+	_Kernel32 = windows.NewLazySystemDLL("kernel32")
+
+	// https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getsystempowerstatus
+	_GetSystemPowerStatus = _Kernel32.NewProc("GetSystemPowerStatus")
+
+	_BatteryFlagCharging        byte = 8
+	_BatteryFlagNoSystemBattery byte = 128
+	_BatteryFlagUnknownStatus   byte = 255
+
+	_BatteryLifePercentUnknown byte = 255
+)
+
+func batteryLevel() (uint8, error) {
+	resp, err := powerStatus()
+	if err != nil {
+		return 100, ErrNotAvailableAPI
+	}
+
+	if resp.BatteryLifePercent&_BatteryLifePercentUnknown > 0 || resp.BatteryFlag&_BatteryFlagNoSystemBattery > 0 {
```

Comment:
```suggestion
	if resp.BatteryLifePercent == _BatteryLifePercentUnknown || resp.BatteryFlag&_BatteryFlagNoSystemBattery > 0 {
```
I think this was an accidental over-generalization. The `BatteryLifePercent` value isn't a flag, so I think it's proper to do direct comparison on it.